programs/voter-stake-registry/tests/program_test/mod.rs

```diff
@@ -17,7 +17,7 @@ use std::{
     str::FromStr,
     sync::{Arc, RwLock},
 };
-pub use utils::*;
+pub use utils::{assert_custom_on_chain_error::AssertCustomOnChainErr, *};
```

Comment:
Do we need to extract `asset_custom`... separately, isn't an asterisk enough?

Reply:
if you remove this import, then wherever it is used, you will have to specify the full path to it